src/keyboard/textinput.js

```diff
@@ -65,14 +65,16 @@ var TextInput = function(parentNode, host) {
         }     
     };
     this.setAriaLabel = function() {
-        var row;
-        if (!host.session)
-            row = 0;
-        else
-            row = host.session.selection.cursor.row;
+        if(host.session && host.renderer.enableKeyboardAccessibility)
+        {
```

Comment:
Doesn't look in line with the code style

Reply:
ah whoops, you're right. Fixed in new commit